Index: .gitattributes
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Auto detect text files and perform LF normalization\r\n* text=auto\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitattributes b/.gitattributes
--- a/.gitattributes	(revision a8c7580ea3b9c2b8fb4a4546b314d8929307d809)
+++ b/.gitattributes	(date 1679501849316)
@@ -1,2 +1,1 @@
-# Auto detect text files and perform LF normalization
-* text=auto
+
